### ISSUE TITLE
Allow use of arbitrary python environments in the python engine

### ISSUE THREAD

**Issue by davidedelvento** [NCAR/VAPOR#3481]
### Is your feature request related to a problem?
I would like to use VAPOR's python engine to do stuff that require me to import many modules. 

### Describe the solution you'd like.
I _might_ be able to do so today by placing all the relevant files in the `VAPOR/share/python` directory. Would that work? Even if it does, it would be unscalable if the various modules become many (which they usually do pretty quickly, in python).

Instead, if the python engine could be initialized with an arbitrary python (or conda) virtualenv that would be awesome.

**Comment by sgpearse** [comment]
Hi @davidedelvento - Just to be clear, this is regarding VAPOR's internal python engine that is used to derive new variables in the GUI, correct?  Not the recently released Python API?

**Comment by davidedelvento** [comment]
Yes, correct, that is what I am thinking (however I am also independently and separately thinking of using the Python API for this reason, but VAPOR's internal python engine would be better for this goal)

Thanks!
